src/Helper/ApplicationHelper.php

```diff
@@ -12,7 +12,19 @@ final class ApplicationHelper
 
     public function getCurrentPHPSUVersion(): string
     {
-        return $this->getPhpSuVersionFromVendor() ?? $this->getPhpSuVersionFromGitFolder() ?? 'development';
+        return $this->getPhpSuVersionFromGlobals() ?? $this->getPhpSuVersionFromVendor() ?? $this->getPhpSuVersionFromGitFolder() ?? 'development';
+    }
+
+    /**
+     * @return string|null
+     */
+    private function getPhpSuVersionFromGlobals()
+    {
+        $pharVersion = '@phpsu_version@';
```

Comment:
how does this return anything besides null? is box auto-replacing this on build?

Reply:
yes

Reply:
:+1: